src/main/java/com/radixdlt/atommodel/Atom.java

```diff
@@ -287,7 +277,7 @@ public final AID getAID() {
 	}
 
 	private AID doGetAID() {
-		return AID.from(getHash(), this.getShards());
```

Comment:
I believe this will break javascript libraries. @alexmedkex will changing the AID format break things on your end?

Reply:
It will affect the Javascript lib, and there is an issue in place for that.  Not really sure how we would remove the sharding concept and still retain the way that AIDs are calculated.

Reply:
right, my thought was that these java/js changes should be done within the same sprint (atomic commit :) so that we dont have nonfunctioning libraries